### ISSUE TITLE
A few questions on PlotAggregate

### ISSUE THREAD

**Issue by cardinLL** [loosolab/TOBIAS#187]
Hi, msbentson,

I encountered some problems when using the PlotAggregate command. I obtained the binding pattern of Stat2 from three separate BINDetect results and wanted to visualize the difference in its bound footprint across different conditions. These are the three plot that I got: 
![Stat2_aggregate](https://user-images.githubusercontent.com/97207965/215836073-e7960235-d8a4-4d0f-be5b-411e14e02232.png)
![Stat2_aggregate2](https://user-images.githubusercontent.com/97207965/215839594-5f772353-2731-4eb2-80bd-5a18d1c3d080.png)
![Stat2_aggregate](https://user-images.githubusercontent.com/97207965/215839685-e26b3d4c-8ff3-42b8-a394-9f4816aad385.png)
comparing to your example picture, two of them are quite different:
![image](https://user-images.githubusercontent.com/97207965/215836291-0d7a6a53-f575-48d4-b4f0-e3442ddc76fe.png)
I wonder if this is problematic or normal. If this is problematic, what might have caused this problem? 
Additionally, I am just a beginner in bioinformatics. So I would like to ask a very fundamental question: what does the y value in an aggregated plot mean and how to interpret an aggregated plot? You can use the third picture that I attach as an example if you wish, since it looks normal. 
I will greatly appreciate any help!! 

Regards, 

Lihao Yang

**Comment by msbentsen** [comment]
Hi Lihao,

Which bigwig files are you using to make the plot with `PlotAggregate`? It looks like it might be the footprint score files (which represent the scores at each position), but the ones from the example are calculated from the `_corrected.bw` files, which represent the Tn5 insertion. 

The y-axis is the mean insertion rate across all the sites in the plot (the upper right corner of each plot shows how many sites there are). So you can imagine a large matrix of centered signals, where we take the column mean for each base. Hope that clears up the question.

**Comment by cardinLL** [comment]
Hi msbentsen, 

Thank you for your response! I figured out that I used the uncorrected bigwig file, which might have caused the problem. Also, I just need a little bit of nudge on interpreting an aggregate plot. For the third picture that I attached, is it appropriate to interpret it as: Stat2 shows more binding sites in ONC1D condition but has deeper footprint in Naive and ONC3D condition. 

Regards, 

Lihao Yang
